CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added new experimental CookAndGenerateSchemaCommandlet that generates required schema during a regular cook.
 - Added OverrideSpatialOffloading command line flag that allows toggling of offloading at launch time.
 - Added an AuthorityIntent component to be used in the future for UnrealGDK code to control loadbalancing.
+- Added support for RPC tracking and for actor property tracking via UE4s Network Profiler. This is a good proxy; Reasons for why it might not be fully accurate:
```

Comment:
1. This markdown doesn't seem to render properly.
2. I think the subpoints should be in the public docs rather than the release notes, and the release note can be something like "Added support for the UE4 Network Profile to measure relative size of RPC and Actor replication data."

Reply:
Removed